src/main/kotlin/logic/usecase/SuggestHighCalorieMealUseCase.kt

```diff
@@ -0,0 +1,44 @@
+package org.example.logic.usecase
+
+
+import org.example.logic.model.Meal
+import org.example.logic.repository.MealsRepository
+
+
+class SuggestHighCalorieMealUseCase(
+    private val mealsRepository: MealsRepository
+) {
+
+
+    fun suggestNameAndDescriptionOfHighCalorieMeal(): Pair<String?, String?> {
+        val suggestionPairMeal = mealsRepository.getAllMeals()
+            .filter(::filterNameAndDescription)
+            .filter(::filterMealByHighCalorie)
+            .map { it.name to it.description }
+            .random()
+        return suggestionPairMeal
+    }
+
+    private fun filterNameAndDescription(meal: Meal): Boolean{
+        return  meal.name!==null && meal.description!=null
+    }
+
+    private fun filterMealByHighCalorie(meal: Meal): Boolean{
+        return  meal.nutrition?.calories!! > 700f
```

Comment:
i dont think using not-null assertion is optimal here 
think of elvis operator or let function
```suggestion
        return  meal.nutrition?.calories!! > 700f
```

Reply:
okay boss